content/source/docs/enterprise/upgrade/index.html.md

```diff
@@ -68,23 +68,44 @@ terraform {
 }
 ```
 
-### 3.e. Reinitialize Terraform again
+### 3.e. Reinitialize Terraform Again
 
 Terraform's `init` command automatically migrates state from the legacy environment to the new workspace when the [backend is reconfigured](../../backends/config.html#changing-configuration). Run the `init` command, and it will ask to migrate the state. Answer `yes`.
 
 ```bash
 terraform init
 ```
 
-## 4. Disable VCS integration in the legacy environment
+## 4. Set Variables in the New Workspace
+
+Open two side-by-side browser windows. In one of them, navigate to the new workspace you created and click on the "Variables" tab. In the other, navigate to the legacy environment you're migrating and view its variables.
+
+Copy the old environment's variables and values to the new workspace, ensuring that all of the values are identical.
```

Comment:
Might be worth calling out that they'll need to re-enter the sensitive variables, as those are not readable from the old workspace.

Reply:
ah good call